docs/docsite/rst/community/development_process.rst

```diff
@@ -180,33 +180,37 @@ The toplevel keys (bugfixes, major_changes, and so on) are defined in the
   Features that were previously deprecated and are now removed.
 
 **bugfixes**
-  Fixes that resolve issues. If there is a specific issue related to this bugfix, add a link in the changelog entry.
+  Fixes that resolve issues.
 
 **known_issues**
   Known issues that are currently not fixed or will not be fixed.
 
-Most changelog entries will be ``bugfixes`` or ``minor_changes``. When writing a changelog entry that pertains to a particular module, start the entry with ``- [module name] -`` and include a link to the related issue if one exists.
+Each changelog entry must contain a link to its issue between parentheses at the end.
+If there is no corresponding issue, the entry must contain a link to the PR itself.
+
+Most changelog entries will be ``bugfixes`` or ``minor_changes``. When writing a changelog entry that pertains to a particular module, start the entry with ``- [module name] -`` and the following sentence with a lowercase letter.
 
 Here are some examples:
 
 .. code-block:: yaml
 
   bugfixes:
-    - win_updates - fixed issue where running win_updates on async fails without any error
+    - win_updates - fixed issue where running win_updates on async fails
```

Comment:
Do we want to use a different example here? The win_updates module is not in ansible/ansible any more.

Reply:
good point, changed to `apt_repository`